### PR TITLE
Icon articles docs

### PR DESCRIPTION
**Pull request by svdimitr**
This pull request is draft, because the Drawer Icons article has no screenshot, since there is a visual bug.

### CODE REVIEW

**Comment by svdimitr** [comment]
Drawer article is updated

File: components/menu/icons.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: Icons
+page_title: Menu for Blazor | Icon
+description: Icons and images in the Menu for Blazor
+slug: menu-icons
+tags: telerik,blazor,menu,icon,iconclass,image
+published: True
+position: 2
```
Comment (marin-bratanov) [inline components/menu/icons.md:8]: Move later in the tree on the left, it is not a very important feature, it should be around templates

File: components/menu/icons.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+---
+
+# Menu Icons
+
+You can put an image, icon class or a font icon for each item in the menu to illustrate its purpose for your end users. To apply them, use the following properties:
+
+* for a font icon, populate the `IconField` parameter of the component or provide an `Icon` property in the data model.
```
Comment (marin-bratanov) [inline components/menu/icons.md:15]: fix this up in the same manner as the Drawer article - including factological explanations here, indentations and field names that clearly denote what the fields hold.

File: components/menu/icons.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        public string CustomIcon { get; set; }
+        public string Image { get; set; }
+        public string CustomIconClass { get; set; }
+    }
+}
+````
+
+>caption The result from the code snippet above
+
+![icons](images/icons.jpg)
```
Comment (marin-bratanov) [inline components/menu/icons.md:76]: the image is blurry

File: components/treeview/icons.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
+---
```
Comment (marin-bratanov) [inline components/treeview/icons.md:1]: same as the menu article

File: components/menu/icons.md (hunk 1 above)
Comment (svdimitr) [reply components/menu/icons.md]: Done with fb6a044a0f7b443962247314e76ac0a28fe459fc

File: components/menu/icons.md (hunk 2 above)
Comment (svdimitr) [reply components/menu/icons.md]: Done with fb6a044a0f7b443962247314e76ac0a28fe459fc

**Pull request merged**